eurotoken/src/main/java/nl/tudelft/trustchain/eurotoken/community/EuroTokenCommunity.kt

```diff
@@ -87,7 +87,7 @@ class EuroTokenCommunity(
     private fun onRollbackRequest(peer: Peer, payload: RollbackRequestPayload) {
         transactionRepository.attemptRollback(peer, payload.transactionHash)
     }
-
+    
```

Comment:
Minor comment: please avoid/remove these small cosmetic changes. They can potentially lead to rebase conflicts with other PRs. I also noticed a few added newlines in the main `README.md` that were not needed.